int64 Frame_id
int32 Object_id
int32[] Grid_size
int32[] Occupancy_Grid
geometry_msgs/Point[] BB_Points
geometry_msgs/Point[] Block_center_Points
